osu.Game/Overlays/Settings/Sections/SkinSection.cs

```diff
@@ -75,92 +70,91 @@ private void load(OsuConfigManager config, [CanBeNull] SkinEditorOverlay skinEdi
                 new ExportSkinButton(),
             };
 
-            skins.ItemUpdated += itemUpdated;
-            skins.ItemRemoved += itemRemoved;
-
             config.BindWith(OsuSetting.Skin, configBindable);
+        }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
 
             skinDropdown.Current = dropdownBindable;
+
+            realmSkins = realmFactory.Context.All<SkinInfo>()
+                                     .Where(s => !s.DeletePending)
+                                     .OrderBy(s => s.Protected)
```

Comment:
If the protected items are to show at the top of the list, then I'm pretty sure this should be
```suggestion
                                     .OrderByDescending(s => s.Protected)
```
When sorting bools, `false` takes precedence as the "zero" value ([see sharplab](https://sharplab.io/#v2:C4LgTgrgdgPgAgJgIwFgBQcAMACOSB0AMgJZQCOA3OungJwAUem+AUgPan0BEANL9lACmAd2wAjNmwA2AbQC62AN7ZgkQT2wAzAIZSAzoOwBffAHkwAE0FgAQgE96Y7AF4AfOICU+AMqCpggGNgRxd3MXwAFTZvVVIAc3oPJI8qDCQGJlYOKG4+DSFRCWl5JS1dAw1VCEMTcytbOwARQT0AwSgLeJC3Tx8/QOCnHvComLAu5JSgA)).

Reply:
Weird, not sure how this works correctly right now in that case.

Reply:
... apparently it never did work. I guess I was always checking after doing an import from the base state of having no skins, where it would look correct?